conda_smithy/lint_recipe.py

```diff
@@ -252,7 +252,11 @@ def lintify(meta, recipe_dir=None, conda_forge=False):
 
     # 10: License should not include the word 'license'.
     license = about_section.get("license", "").lower()
-    if "license" in license.lower() and "unlicense" not in license.lower():
+    if (
+        "license" in license.lower()
+        and "unlicense" not in license.lower()
```

Comment:
There's also 2 licenses `BSD-3-Clause-no-nuclear`

Reply:
Huh? I am sorry but I don't follow the comment. Do you have the time to explain a bit?

Reply:
For example https://github.com/spdx/license-list-data/blob/master/text/BSD-3-Clause-No-Nuclear-License-2014.txt. Wouldn't this license get the error too?

Reply:
Ohhhhhhh thank you. I will fix it to simply check for the single word license since I think that is the intended lint.

Reply:
OK @isuruf . I put in a check for `-license` as well since all SPDX things will look like that.

Reply:
I just read my first comment again and I had forgotten `license` there. Sorry about the confusion.

Reply:
no worries! thank you for the code review!